src/content/docs/products/splashkit/SplashKit Tutorials/Onboarding/01-windows-installation.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Windows Installation Guide
+description: Use this template to create a proposal for a new SplashKit tutorial.
+sidebar:
+  order: 1
+  label: 1. Windows Installation
+  attrs:
+    class: windows
+---
+
+This is a guide on how to install Splashkit on Windows, along with Visual Studio Code and the necessary extensions required to run Splashkit. This guide is intended to get you set up and ready to contribute to the SplashKit team if you're a Windows user. This installation guide was done on Windows 10.
+
+## Install msys2
+
+### Download msys2
+
+To start, head to the official [msys2 website](https://www.msys2.org/) and download the installer.
+
+![msys2](/splashkit/windows-installation-fig1.png)
+
+### Install msys2
+
+When the installer is downloaded, run it, you will get a pop up that looks like the following:
+
+![msys2](/splashkit/windows-installation-fig2.png)
+
+You will need to click `more info` then click run. The following shows the screenshots of each step and the options selected:
```

Comment:
This may not necessarily be the same for everyone. For instance, I don't run Windows Defender and don't get that warning. I would recommend generalising this and prompting the user with something like:

"Your computer security may prevent the application from starting... <actions on from here>"

Then say, 

"For example, if Windows Defender prevents the application from starting... <actions on from here>. See screenshot below".

Reply:
Fixed, I assumed majority of people would use windows defender by default.